assets/js/translator.js

```diff
@@ -89,6 +89,13 @@ if(modeEnabled('translation')) {
                     $('div#translateText').fadeIn('fast', function () {
                         synchronizeTextareaHeights();
                     });
+                    if($('#detect').hasClass('originDetect') === true) {
+                        $('#srcLang1').removeClass('active');
```

Comment:
I think `.srcLang` would work here and below.

Reply:
@sushain97 ok, changing.